### PR TITLE
Adding Akismet autoconnection

### PR DESCRIPTION
**Pull request by pauarge**
This PR extends Jetpack Connection Pilot capability to add autoconnection to Akismet. If a Jetpack connection is active, the system will try to connect automatically to Akismet

## Description

This PR extends Jetpack Connection Pilot capability to add autoconnection to Akismet. If a Jetpack connection is active, the system will try to connect automatically to Akismet.

### Akismet auto connection

Connection Pilot is now capable of autoconnecting to Akismet.

## Checklist

Please make sure the items below have been covered before requesting a review:

- [ ] This change works and has been tested locally (or has an appropriate fallback).
- [ ] This change works and has been tested on a Go sandbox.
- [ ] This change has relevant unit tests (if applicable).
- [ ] This change has relevant documentation additions / updates (if applicable).
- [x] I've created a changelog description that aligns with the provided examples.

## Steps to Test

1. Check out PR.
2. Go to a Sandbox and launch a CLI.
3. Run the `reconnect()` routine from Connection Pilot.

### CODE REVIEW

File: vip-jetpack/connection-pilot/class-jetpack-connection-controls.php [hunk 1]
```diff
@@ -130,6 +130,15 @@ public static function connect_site( $skip_connection_tests = false, $disconnect
 		return self::jetpack_is_connected();
 	}
 
+	/**
+	 * Connect a site to Akismet.
+	 *
+	 * Uses Akismet's function to connect Akismet using the Jetpack. An active Jetpack connection is required on the site.
+	 */
+	public static function connect_akismet() {
+		return \Akismet_Admin::connect_jetpack_user();
```
Comment (rinatkhaziev) [inline vip-jetpack/connection-pilot/class-jetpack-connection-controls.php:139]: Should we add a `class_exists` check here, just to be safe?

File: vip-jetpack/connection-pilot/class-jetpack-connection-pilot.php [hunk 2]
```diff
@@ -149,12 +149,20 @@ public function reconnect() {
...
 			$this->send_alert( 'Jetpack was successfully (re)connected!' );
 
+			$akismet_connection_attempt = Connection_Pilot\Controls::connect_akismet();
+
+			if ( ! $akismet_connection_attempt ) {
+				$this->send_alert( 'Alert: Could not connect Akismet automatically.' );
+				return;
+			}
+
+			$this->send_alert( 'Akismet was successfuly connected!' );
```
Comment (WPprodigy) [inline vip-jetpack/connection-pilot/class-jetpack-connection-pilot.php:164]: I think we can do without this being an alert. Just knowing when it fails seems sufficient.

**Review by rinatkhaziev — APPROVED** [review]
Thanks, looks good!

**Comment by rinatkhaziev** [comment]
good catch!

**Review by rinatkhaziev — APPROVED** [review]

**Pull request merged**